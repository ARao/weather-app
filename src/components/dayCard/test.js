import React from 'react';
import DayCard from './index'
import { shallow } from 'enzyme'
import day from '../../__testData__/day.json'


describe('DayCard', () => {

    it('++++ render Daycard with index 0', () => {
        let wrapper = shallow(<DayCard day={day} index={0} />);
        expect(wrapper.length).toEqual(1);
    });

    it('++++ render daycar  with index 1', () => {
        let wrapper = shallow(<DayCard day={day} index={1} />);
        expect(wrapper.length).toEqual(1);
    });

    it('++++ snapshot match', () => {
        let wrapper = shallow(<DayCard day={day} index={1} />);
        expect(wrapper).toMatchSnapshot();
    });
});
