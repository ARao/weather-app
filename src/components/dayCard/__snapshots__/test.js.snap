// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayCard ++++ snapshot match 1`] = `
<Link
  className="col-md-2 p-4 "
  key="2019-01-10--1"
  replace={false}
  to="detail/1"
>
  <small>
    2019-01-10
  </small>
  <div>
    <img
      alt="weather icon"
      src="//cdn.apixu.com/weather/64x64/day/176.png"
    />
  </div>
  <small>
    Patchy rain possible
  </small>
  <div>
    <small
      className="pr-3 "
    >
      3.8
    </small>
    <small>
      5
    </small>
  </div>
</Link>
`;
